packages/design-system/src/components/label.tsx

```diff
@@ -0,0 +1,78 @@
+import type { LabelHTMLAttributes, ReactNode, Ref } from "react";
+import { forwardRef } from "react";
+import { styled } from "../stitches.config";
+import { theme } from "../stitches.config";
+
+const StyledLabel = styled("label", {
```

Comment:
The thing is the "label" is too generic for this component, because this component is specific to style panel PropertyName.

Maybe this should be renamed in figma, because this seems super misleading.  I started conversation in figma comments about this, lets wait for Taylor to comment

Reply:
Totally agree

Reply:
After talking with Taylor we agreed to keep calling it label because its having general-purpose variants + special ones we use in style panel

This is basically what we do with all design system components to avoid having to create a separate design system for designer